packages/wdio-cucumber-framework/src/cucumberEventListener.ts

```diff
@@ -253,15 +256,24 @@ export default class CucumberEventListener extends EventEmitter {
 
         const doc = this._gherkinDocEvents.find(gde => gde.uri === scenario?.uri)
 
+        if (this._gherkinDocEvents.length > 1) {
```

Comment:
This should have a comment as to why we have an if statement here

Reply:
Outdated